migrations/migration-runner.js

```
@@ -17,5 +17,5 @@ function setUpServer () {
 setUpServer();
 
 // Replace this with your migration
-var processUsers = require('./new_stuff');
-processUsers();
+var syncAllChallenges = require('./challenges/sync-all-challenges');
+syncAllChallenges();

```

here probably you want to add a `.catch` otherwise errors will be swallowed